src/android/device.rs

```diff
@@ -40,15 +86,102 @@ impl Reportable for ApkBuildError {
     }
 }
 
+#[derive(Debug)]
+pub enum AabBuildError {
+    BuildFailed(bossy::Error),
+}
+
+impl Reportable for AabBuildError {
+    fn report(&self) -> Report {
+        match self {
+            Self::BuildFailed(err) => Report::error("Failed to build AAB", err),
+        }
+    }
+}
+
+#[derive(Debug)]
+pub enum ApksBuildError {
+    CleanFailed(std::io::Error),
+    BuildFromAabFailed(bossy::Error),
+    InvalidUtf8InAabPath(PathBuf),
+    InvalidUtf8InApksPath(PathBuf),
+}
+
+impl Reportable for ApksBuildError {
+    fn report(&self) -> Report {
+        match self {
+            Self::CleanFailed(err) => Report::error("Failed to clean old APKS", err),
+            Self::BuildFromAabFailed(err) => Report::error("Failed to build APKS from AAB", err),
+            Self::InvalidUtf8InApksPath(path) => Report::error(
+                "apks path {:?} contained invalid utf-8",
```

Comment:
```suggestion
                "APKS path {:?} contained invalid utf-8",
```
applies to variant below as well